packages/ses/src/compartment-shim.js

```diff
@@ -120,6 +127,32 @@ export const CompartmentPrototype = {
     return weaksetHas(knownScopeProxies, value);
   },
 
+  /**
+   * @param {Object} [options]
+   * @param {boolean} [options.sloppyGlobalsMode]
+   * @param {Object} [options.__moduleShimLexicals__]
+   */
+  /* eslint-disable-next-line no-underscore-dangle */
+  __makeScopeProxy__(options = {}) {
```

Comment:
The body of this looks like it’s redundant with part of `performEval`. Let’s make most of this a utility function used by both `performEval` and `__makeScopeProxy__`. It can be in the `evaluate.js` module with `performEval`.